### PR TITLE
add testcase with api/errors/errors_test.go

### PR DESCRIPTION
**Pull request by chchliang**
Signed-off-by: chchliang <chen.chuanliang@zte.com.cn>

Add testcase with api/errors/errors.go

### CODE REVIEW

File: api/errors/errors_test.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+	errmsg = newError("testRequestNotFoundError")
+	nerr := NewRequestNotFoundError(err.error)
+	apierr, a = nerr.(apiError)
+	if !a {
+		t.Fatal("excepted err is apiError type")
+	}
+	assert.Equal(t, http.StatusNotFound, apierr.statusCode)
+
+	errmsg = newError("testRequestConflictError")
```
Comment (AkihiroSuda) [inline api/errors/errors_test.go]: unused?

Comment (chchliang) [inline api/errors/errors_test.go]: @AkihiroSuda 
I don't understand what you mean.

Comment (vdemeester) [inline api/errors/errors_test.go]: @chchliang @AkihiroSuda meant that this `errmsg` variable is not used below, so either something is missing or it should be there 👼

Comment (chchliang) [inline api/errors/errors_test.go]: @vdemeester @AkihiroSuda 
I have changed, thanks!

**Comment by chchliang** [comment]
@vdemeester 
I think  that this PR has nothing to do with the `windowsRS1 — Jenkins build Docker-PRs-WoW-RS1 13309 has failed`

error messages

09:12:55 INFO: Commit hash is f157b5460
09:12:55 INFO: Nuke-Everything...
09:12:55 INFO: Container count on control daemon to delete is 0
12:32:51 Build timed out (after 200 minutes). Marking the build as failed.
12:32:52 Set build name.
12:32:52 New build name is '#32877'
12:32:52 Variable with name 'BUILD_DISPLAY_NAME' already exists, current value: '#32877', new value: '#32877'
12:32:52 Build was aborted
12:32:52 [PostBuildScript] - Execution post build scripts.
12:32:52 Notifying endpoint 'HTTP:https://leeroy.dockerproject.org/notification/jenkins'
12:32:52 Finished: FAILURE

**Review by AkihiroSuda — APPROVED** [review]
LGTM

cc @vdemeester

**Review by vdemeester — APPROVED** [review]
LGTM 🐯

**Pull request merged**